autoscale/src/main/resources/application.yml

```diff
@@ -64,6 +64,8 @@ cb:
       connect.timeout.seconds: 245
       read.timeout.seconds: 245
       write.timeout.seconds: 245
+      commands:
+        deployClientConfig.interrupt.timeout.seconds: 120
```

Comment:
I think we dont use camel cases here

Reply:
there are many camel case usages in the config, also i would not separate the command name